data_explore.ipynb

```diff
@@ -0,0 +1,91 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#This is a script to explore the inherent biases in a given data set\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from aif360.metrics import BinaryLabelDatasetMetric\n",
+    "from aif360.algorithms.preprocessing.optim_preproc_helpers.data_preproc_functions\\\n",
+    "            import load_preproc_data_adult"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/krishnanamboothiri/opt/anaconda3/envs/debias/lib/python3.7/site-packages/aif360/datasets/standard_dataset.py:142: FutureWarning: outer method for ufunc <ufunc 'equal'> is not implemented on pandas objects. Returning an ndarray, but in the future this will raise a 'NotImplementedError'. Consider explicitly converting the Series to an array with '.array' first.\n",
+      "  df[label_name]))\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Load dataset, specifying protected attribute, and split dataset into train and test\n",
+    "dataset_orig = load_preproc_data_adult(['sex'])\n",
+    "dataset_orig_train, dataset_orig_test = dataset_orig.split([0.7], shuffle=True)\n",
+    "privileged_groups = [{'sex': 1}] # White\n",
+    "unprivileged_groups = [{'sex': 0}] # Not white"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "#### Original training dataset\n",
+      "Difference in mean outcomes between unprivileged and privileged groups = -0.196235\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Check bias metric on original training dataset\n",
+    "metric_orig_train = BinaryLabelDatasetMetric(dataset_orig_train,\n",
+    "                                             unprivileged_groups=unprivileged_groups,\n",
+    "                                             privileged_groups=privileged_groups)\n",
+    "print((\"#### Original training dataset\"))\n",
+    "print(\"Difference in mean outcomes between unprivileged and privileged groups = %f\" % metric_orig_train.mean_difference())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
Please put your code into .py scripts as soon as possible.  The git diffs for notebooks render in JSON which is hard to read.